packages/ui/src/components/atoms/button.tsx

```
@@ -3,41 +3,70 @@
+  disabled?: boolean;
+}): string | undefined => {
+  if (disabled) return "";
+
+  switch (variant) {
+    case "secondary":
+    case "tertiary":
+      if (statusColor === "success") return "text-success-500";
+      if (statusColor === "warning") return "text-warning-500";
+      if (statusColor === "error") return "text-error-500";
+      break;
+    case "primary":
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)